test/Release/MysqlReleaseGatewayTest.php

```diff
@@ -0,0 +1,161 @@
+<?php
+/**
+ * @author: Ryan Wood <ryanw@clearbooks.co.uk>
+ * @created: 10/08/15
+ */
+
+namespace Clearbooks\LabsMysql\Release;
+
+
+use Clearbooks\Labs\Release\Release;
+use Doctrine\DBAL\Configuration;
+use Doctrine\DBAL\Connection;
+use Doctrine\DBAL\DriverManager;
+
+class MysqlReleaseGatewayTest extends \PHPUnit_Framework_TestCase
+{
+    /**
+     * @var MysqlReleaseGateway
+     */
+    private $gateway;
+
+    /**
+     * @var Connection
+     */
+    private $connection;
+
+    public function setUp()
+    {
+        parent::setUp();
+
+        $connectionParams = array(
```

Comment:
Hm shouldn't the DB connection details be stored somewhere else? If we do it like this, we would need to provide all this in every separate test class, wouldn't we?